CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 * [Breaking] Rename `yarpc.CreateThriftServiceFunc` to `yarpc.ServiceCreateFunc`
   as it is not thrift-specific.
 * Report version metrics for company-wide version usage information.
+* [Breaking] Use default config name as service name and allow both service name and module name
```

Comment:
Sure, it's breaking from previous change but not from beta1, I can remove. Aiden has been getting me in the habit of doing this. Why does it not belong in the changelog?
There is difference in functionality, users can specify custom module and service name in options.